src/utils/time.ts

```diff
@@ -229,9 +188,10 @@ export function formatDateToWeekday(utcSeconds: number) {
  * @see https://date-fns.org/docs/formatRelative
  */
 export const formatUtcSecondsRelativeToNow = (utcSeconds: number) => {
-  return formatRelative(utcSeconds * ONE_SECOND_IN_MILLISECONDS, Date.now());
+  return formatRelative(fromUnixTime(utcSeconds), Date.now());
 };
 
+// @debt get rid of ONE_SECOND_IN_MILLISECONDS and use date-fns function
 export const normalizeTimestampToMilliseconds = (timestamp: number) => {
   const isTimestampInMilliSeconds = timestamp > SECONDS_TIMESTAMP_MAX_VALUE;
```

Comment:
This `@debt` comment seems wrong? There is no `ONE_SECOND_IN_MILLISECONDS` here.

`SECONDS_TIMESTAMP_MAX_VALUE` is a sort of edgecase/hacky thing that might not have a correlation in `date-fns`/similar, so may not be worth adding an `@debt` comment here.

```suggestion
export const normalizeTimestampToMilliseconds = (timestamp: number) => {
  const isTimestampInMilliSeconds = timestamp > SECONDS_TIMESTAMP_MAX_VALUE;
```

Reply:
`ONE_SECOND_IN_MILLISECONDS` is used in the return statement, moved the `@debt` comment there: https://github.com/sparkletown/sparkle/pull/1302/commits/61c8d31c7eecfeae9b1debbdc10b22f31a630afe